### PR TITLE
Update python package versions

### PR DESCRIPTION
**Pull request by PMax5**

### CODE REVIEW

**Review by diocas — APPROVED** [review]

**Review by diocas — COMMENTED** [review]
Shall we revert https://github.com/swan-cern/jupyter-images/commit/03376584f0ce2e10c8cb6dd4411eb2a69db34755 in this PR as well?

**Comment by etejedor** [comment]
> Shall we revert https://github.com/swan-cern/jupyter-images/commit/03376584f0ce2e10c8cb6dd4411eb2a69db34755 in this PR as well?

Yes, we had already discussed this with @PMax5 . He'll add the revert commit first and then his current commit on top.

File: swan/Dockerfile [hunk 1]
```diff
@@ -64,9 +64,11 @@ RUN dnf -y install HEP_OSlibs-9.1.0-2.el9
 # Switch back to jovyan to avoid accidental container runs as root
 USER ${NB_UID}
 
+RUN pip install --no-deps --no-cache-dir \
```
Comment (diocas) [inline swan/Dockerfile:67]: Not available in conda forge? Maybe document it so that we know why is not bellow.

**Review by diocas — CHANGES_REQUESTED** [review]

File: swan/Dockerfile (hunk 1 above)
Comment (PMax5) [reply swan/Dockerfile:67]: Done

**Review by diocas — APPROVED** [review]

**Pull request merged**